sdk/metricsadvisor/azure-ai-metricsadvisor/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### Other Changes
 - Python 2.7 is no longer supported. Please use Python version 3.6 or later.
```

Comment:
> Python 2.7 is no longer supported.
We may remove the second sentence as it is reflected in the next list item. Great job, my friend.

Reply:
Ah looks like we haven't released this package in a bit. Personally I'd combine the bullet points, but call out the versions we dropped like. "Python 2.7 and 3.6 no longer supported...".

Reply:
Thanks for your feedback. I'll take that into consideration soon.